config/routes.rb

```diff
@@ -229,6 +229,14 @@
     resources :users, only: [:index]
     resources :permission_template_accesses, only: :destroy
     resource 'stats', only: [:show]
+    # TODO: from the analytics sprint, admin 'reports' will eventually replace 'stats' above
+    namespace :reports do
+      namespace :works do
+        get 'status', to: 'work_status_report#show'
```

Comment:
@sethaj Are you expecting to have more routes to each of these controllers? Instead of a controller for each action, would it make sense to have one controller called Report(or similar) with actions for status, attributes, and activity. It could reduce the number of nested files and keep similar actions grouped together. If you expect these controller to get complicated then yeah they should probably be separate.